pcluster/pcluster_stack.py

```diff
@@ -71,13 +72,18 @@ def __init__(self, scope: cdk.Construct, id: str, **kwargs) -> None:
             path='scripts/post_install_script.sh'
         )
 
+        # Upload parallel cluster post_install_script to that bucket
+        pcluster_config_script = assets.Asset(self, 'PclusterConfigScript',
+            path='scripts/config.ini'
+        )
+
         # Setup CloudTrail
         cloudtrail.Trail(self, 'CloudTrail', bucket=cloudtrail_bucket)
 
         # Create a Cloud9 instance
         # Cloud9 doesn't have the ability to provide userdata
         # Because of this we need to use SSM run command
-        cloud9_instance = cloud9.Ec2Environment(self, 'Cloud9Env', ec2_environment_name='ResearchWorkspace', vpc=vpc, instance_type=ec2.InstanceType(instance_type_identifier='c5.large'))
+        cloud9_instance = cloud9.Ec2Environment(self, 'ResearchWorkspace', vpc=vpc, instance_type=ec2.InstanceType(instance_type_identifier='c5.large'))
```

Comment:
What name does it get now?

Reply:
The old name was "ResearchWorkspace". The new name is "ResearchWorkspaceXXXXXXX-XXXXXXXXXX"; e.g., ResearchWorkspace61444B30-mcR8ISa4wAkI, where the suffix, mcR8ISa4wAkI, is unique to each stack deployment.